src/app/relay/CreateCommentMutation.js

```diff
@@ -11,8 +11,8 @@ class CreateCommentMutation extends Relay.Mutation {
   getFatQuery() {
     let query = '';
     switch (this.props.parent_type) {
-    case 'source':
-      query = Relay.QL`fragment on CreateCommentPayload { commentEdge, source { annotations } }`;
+    case 'project_source':
+      query = Relay.QL`fragment on CreateCommentPayload { commentEdge, project_source { source { log, log_count } } }`;
```

Comment:
Should be related to source, checkout this note https://mantis.meedan.com/view.php?id=6174#c22350